packages/web/docs/src/pages/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-title: Schema Registry and Observability - GraphQL Hive
+title: Open-source GraphQL Federation Platform
 description:
-  Open-source GraphQL platform for managing APIs with federation and stitching, preventing breaking
-  changes, and optimizing performance. Self-host or deploy in the cloud for complete data control.
+  Fully Open-source schema registry, analytics and gateway for GraphQL federation and other GraphQL
```

Comment:
we should also update https://github.com/graphql-hive/platform/blob/fb53f135ba59864b89e4c830414a887c156a86aa/packages/web/app/src/components/ui/meta.tsx#L3

Reply:
This one is not important as it's app.graphql-hive.com, not the landing page, so not even indexed.

https://github.com/graphql-hive/platform/pull/5865